packages/core/android/build.gradle

```diff
@@ -161,6 +164,22 @@ dependencies {
   testImplementation "com.github.xgouchet.Elmyr:jvm:1.3.1"
   testImplementation "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0"
   testImplementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
+  unmock 'org.robolectric:android-all:4.4_r1-robolectric-r2'
+}
+
+unMock {
+  keep("android.os.BaseBundle")
+  keep("android.os.Bundle")
+  keep("android.os.Parcel")
+  keepStartingWith("com.android.internal.util.")
+  keepStartingWith("android.util.")
+  keep("android.content.ComponentName")
+  keep("android.os.Looper")
+  keep("android.os.MessageQueue")
+  keep("android.os.SystemProperties")
+  keep("android.view.Choreographer")
+  keep("android.view.DisplayEventReceiver")
```

Comment:
I think we probably need only those to make `Choreographer` testable. Can you remove the rest (and `SystemProperties` probably) and see if it still works?

Reply:
Actually `SystemProperties` is needed